### ISSUE TITLE
"Unknown type 'String'" error in GraphQL request body editor

### ISSUE THREAD

**Issue by lkaratun** [a85/POSTMan-Chrome-Extension#8657]
**Describe the bug**
Getting an error `Unknown type 'String'` in GraphQL request body editor

**To Reproduce**
1. Create a new endpoint, go to body editor
2. Change body type to "GraphQL"
3. Enter the following in the "Query" panel:
```
mutation createAd ($data: String!) {
    createAd (data: $data) {
        id
        description
    }
}
```

4. See red squiggly line with error `Unknown type 'String'`

**Expected behavior**
No error

**Screenshots**
![image](https://user-images.githubusercontent.com/17469132/84614351-d7131900-ae93-11ea-937e-9b94a0860816.png)

**App information (please complete the following information):**
 - App Type Native App
 - Postman Version 7.26.0
 - OS: Windows 10

**Comment by eladcandroid** [comment]
I'm facing the same problem, also with predefined types from GraphQL schema

**Comment by marcoaleixo** [comment]
same problem here

**Comment by middiu** [comment]
same problem for me...

**Comment by srinivasskc** [comment]
I see the same problem. But it does not impact my request to proceed.

**Comment by coro101** [comment]
I imported the schema, and there is the same problem. However, auto-completion works well.

**Comment by benjaminaudet** [comment]
I have the same problem. It's probably a parsing problem, because the request is working, it was not doing that in the previous version (7.25)

**Comment by pankajk07** [comment]
Hi all, we have fixed this as part of `v7.28` release. Please update the app and try it out. Do let us know if you come across any other issues.
